### PR TITLE
fix(account-navigation): post-merge fixes

### LINKED ISSUE #3607: New account navigation nitpicks
**Describe the bug**

There are some problems with new Account navigation design. Please see video/screenshots

 - 1 - When leaving account navigation to Settings it takes some time to replace `< back` button to General/Device/Coins tabs
 - 2 - After you send transaction navigation won't reappear and you will only see `< back ` is it possible to change this behavior ?
 - 3 - Account Detail header is doubled / Card title + Navigation tab name/ 

**Info:**
 - Suite version: web 21.4.1 (3ed454e4e81b72482a9ad780c5cce9846cb29fb1)
 - Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:87.0) Gecko/20100101 Firefox/87.0
 - OS: Linux x86_64
 - Screen: 2560x1440
 - Device: model T 2.3.4 regular

**Screenshots:** 
 - 1

https://user-images.githubusercontent.com/31506317/113839719-39753700-9790-11eb-9e83-0ff34283eafc.mp4

 - 2

image

 - 3 

image

### PR DESCRIPTION
**Pull request by tomexx**
fixes #3607

1. Back button is visible in Settings for a short time after coming from Send/Receive etc. - **I was not able to reproduce this**
2. User is redirected to Overview after signing the transaction
3. Header in detail is not duplicated anymore
4. Active tab states in settings fixed
5. Padding added to tab content
6. Tabs vs Back button height fixed

### CODE REVIEW

File: packages/suite/src/actions/wallet/sendFormActions.ts [hunk 1]
```diff
@@ -259,7 +260,9 @@ export const signTransaction = (
     const decision = await dispatch(modalActions.openDeferredModal({ type: 'review-transaction' }));
     if (decision) {
         // push tx to the network
-        return dispatch(pushTransaction());
+        const result = await dispatch(pushTransaction());
+        dispatch(routerActions.goto('wallet-index', undefined, true));
```
Comment (szymonlesisz) [inline packages/suite/src/actions/wallet/sendFormActions.ts]: this action is shared between send/rbf/invity forms so redirection should be done from send component/hook instead.
another important fact is that there should be a result check before redirection, there could be an error (failed to send...)

move it to `useSendForm` hook [sign method](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/hooks/wallet/useSendForm.ts#L224) pls

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

**Comment by tsusanka** [comment]
Could we finish this today so I can pass a new release candidate? :)

**Comment by tomexx** [comment]
> Could we finish this today so I can pass a new release candidate? :)

I had some troubles with updating unit tests, but @szymonlesisz helped me with that. It should be ready to go now. 👍

**Review by goodhoko — APPROVED** [review]

**Review by szymonlesisz — APPROVED** [review]
approving changes in send, didn't test actual UI

**Pull request merged**